### ISSUE TITLE
ValueError on startup

### ISSUE THREAD

**Issue by borpin** [DurgNomis-drol/ha_toyota#148]
**Describe the bug**
The following error is seen on startup
```
ValueError: Sensor sensor.aygo_current_month_statistics has device class None, state class measurement unit km and suggested precision None thus indicating it has a numeric value; however, it has the non-numeric value: unavailable (<class 'str'>)
2023-07-01 20:53:27.269 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up toyota platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/sensor/__init__.py", line 581, in state
    numerical_value = float(value)  # type:ignore[arg-type]
ValueError: could not convert string to float: 'unavailable'
```
**To Reproduce**
Startup HA

**Expected behavior**
No error in log when sensor in unavailable.

**Screenshots**
![image](https://github.com/DurgNomis-drol/ha_toyota/assets/1322749/6d2f73fd-871c-493c-b8f1-d5fe97e8ed5e)

**Additional context**
nil

**Comment by Szeryff** [comment]
I have the same for monthly and weekly statistics

**Comment by CM000n** [comment]
We recently released version 2.0.0 of the Custom Component with major changes and support for the new Mytoyota API.
As we unfortunately cannot provide support for Custom Component versions < v2.0.0, I hereby close your issue.
If the issue still applies to the current version, you are of course free to open a new issue. 😊
